components/download_images/fondant_component.yaml

```diff
@@ -3,13 +3,13 @@ description: Component that downloads images based on URLs
 image: ghcr.io/ml6team/download_images:dev
 
 consumes:
-  images:
+  image:
```

Comment:
I assume this will break the other pipelines using this component. So we should either update them all, or stick to `images`. 

I would propose to stick to `images` since plural is the standard used in our current example pipelines and components (`text` is both singular and plural), and I think it sounds most natural.